src/clients/cassandra/CassandraConnector.js

```diff
@@ -1,19 +1,60 @@
 'use strict';
 
 const Promise = require('promise');
+const cassandra = require('cassandra-driver');
+const asyncEachLimit = require('async/eachLimit');
+const _ = require('lodash/array');
```

Comment:
PS: if all we need is just `_.chunk`, we might as well make our own utility for that since it can be done in ~5 lines. There's a good [post on SO](https://stackoverflow.com/a/10456644/3817588) that outlines a few approaches.